source/MRViewer/ImGuiMenu.cpp

```diff
@@ -1538,6 +1543,27 @@ bool ImGuiMenu::drawDrawOptionsCheckboxes_( const std::vector<std::shared_ptr<Vi
         someChanges |= make_visualize_checkbox( selectedVisualObjs, "Contour", LabelVisualizePropertyType::Contour, viewportid );
         someChanges |= make_visualize_checkbox( selectedVisualObjs, "Leader line", LabelVisualizePropertyType::LeaderLine, viewportid );
     }
+    if ( allIsObjVoxels )
+    {
+        bool atLeastOneDual = false;
+        bool allDual = true;
+        for ( const auto& data : selectedVisualObjs )
+        {
+            bool isThisDual = data && static_cast<ObjectVoxels*>( data.get() )->getDualMarchingCubes();
+            atLeastOneDual = atLeastOneDual || isThisDual;
+            allDual = allDual && isThisDual;
+        }
+        allDual = allDual && atLeastOneDual;
+
+        bool checked = atLeastOneDual;
+        const bool res = UI::checkboxMixed( "Dual Marching Cubes", &checked, !allDual && atLeastOneDual );
+        if ( checked != atLeastOneDual )
+        {
+            for ( const auto& data : selectedVisualObjs )
+                if ( data )
+                    static_cast<ObjectVoxels*>( data.get() )->setDualMarchingCubes( checked );
```

Comment:
This operation does not look fast, do we need to add context menu option for it? (we can make checkbox in ChangeIso plugin)

Reply:
Removed